src/test/java/bio/terra/common/fixtures/ConnectedOperations.java

```diff
@@ -764,27 +810,53 @@ public void teardown() throws Exception {
             // Order is important: delete all the snapshots first so we eliminate dependencies
             // Then delete the files before the datasets
             for (String snapshotId : createdSnapshotIds) {
-                deleteTestSnapshot(snapshotId);
+                try {
+                    deleteTestSnapshot(snapshotId);
+                } catch (Exception ex) {
+                    logger.info("CLEANUP ERROR! Error deleting snapshot. SnapshotId: {}", snapshotId);
+                }
             }
 
             for (String[] fileInfo : createdFileIds) {
-                deleteTestFile(fileInfo[0], fileInfo[1]);
+                try {
+                    deleteTestFile(fileInfo[0], fileInfo[1]);
+                } catch (Exception ex) {
+                    logger.info("CLEANUP ERROR! Error deleting file. FileId: {}", fileInfo[0]);
+                }
             }
 
+            logger.info("Cleanup Tracking: {} datasets to be removed.", createdDatasetIds.size());
             for (String datasetId : createdDatasetIds) {
-                deleteTestDataset(datasetId);
+                logger.info("Cleanup Tracking: Dataset to be deleted {}", datasetId);
+                try {
+                    deleteTestDataset(datasetId);
+                } catch (Exception ex) {
+                    logger.info("CLEANUP ERROR! Error deleting dataset. DatasetId: {}", datasetId);
+                }
```

Comment:
I still have yet to find a log output that actually logs the "CLEANUP ERROR", but everything seems to be much more consistently passing when this is added.